docs/tutorials/autox/autox.md

```diff
@@ -0,0 +1,201 @@
+---
+template: main.html
+---
+
+# AutoX: Automatic Experiments
+
+AutoX, short for "automatic experiments", allows Iter8 to detect changes to your Kubernetes resources objects and automatically start new experiments, allowing you to test your services as soon as you release a new version.
+
+![AutoX](images/autox.png)
+
+Releasing a new version of an application typically involves the creation of new Kubernetes resource objects and/or updates to existing ones. AutoX can be configured to watch for such changes and automatically launch new experiments. You can configure AutoX with multiple experiment groups and, for each group, specify the Kubernetes resource object that AutoX will watch and one or more experiments to be performed in response to new versions of this object. 
+
+Let us now see this in action using a Kubernetes HTTP service and configuring AutoX so that whenever a new version of the service is released, AutoX will start a new HTTP performance test that will validate if the service meets latency and error-related requirements.
+
+***
+
+???+ warning "Before you begin"
+    1. Try [your first experiment](../../getting-started/your-first-experiment.md). Understand the main [concepts](../../getting-started/concepts.md) behind Iter8 experiments.
+
+## Setup Kubernetes cluster with ArgoCD
+
+AutoX uses [Argo CD](https://argo-cd.readthedocs.io), a popular continuous delivery tool, in order to launch the experiments.
+
+A basic install of Argo CD can be done as follows:
+
+```bash
+kubectl create namespace argocd
+kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+```
+
+See [here](https://argo-cd.readthedocs.io/en/stable/getting_started/#1-install-argo-cd) for more information about installation.
+
+## Deploy application
+
+Now, we will create an `httpbin` deployment and service.
+
+```bash
+kubectl create deployment httpbin --image=kennethreitz/httpbin --port=80
+kubectl expose deployment httpbin --port=80
+```
+
+## Apply version label
+
+Next, we will assign `httpbin` deployment the `app.kubernetes.io/version` label (version label). AutoX will relaunch experiments when the version label changes.
```

Comment:
Instead of the next two paragraphs, perhaps we can use ... 
***
Next, we will assign `httpbin` deployment the `app.kubernetes.io/version` label (version label). AutoX will launch experiments only when this label is present on your application resource object. It will relaunch experiments whenever this version label is modified.